### ISSUE TITLE
Sort validation messages based on fields of ActionBean

### ISSUE THREAD

**Issue by iluvtr** [StripesFramework/stripes#19]

**Comment by vankeisb** [comment]
Do you mean when using `s:errors` ? Or do you use ValidationErrors by API calls yourself to list and display the errors ?

**Comment by iluvtr** [comment]
Yes, for example when I want to give a JSON response with the validation errors, the messages are with random sort. Some time ago I code a hack to order the errors given the fields of the actionbean but i think it's something the framework should do.

**Comment by vankeisb** [comment]
Ok, what's the comparator ? Lexically on the field name ?
